docs/whatsnew.md

```diff
@@ -0,0 +1,35 @@
+---
+id: whatsnew
+title: Clay Recaps
+sidebar_label: Clay Recaps
+---
+
+# What's Heating!
+
+Since the beginning of this year Clay Platform team have been working on a couple of things and we wanted to share them with you. 
+
+Here a few of our favorites:
+
+### Clay Docs
+
+We finally have Clay Documentation in one place.
+
+### Clay Starter
+
+[Clay Starter](https://github.com/clay/clay-starter/) is the starting point where you can grasp the capabilities of Clay CMS in a friendly environment that is flexible enough to adapt to your needs.
+
+### Pyxis
+
+Pyxis is an Asset Management System built in NodeJs that uses Sharp for fast image processing.
+
+### Amphora Authentication
+
+A module to offer autenthication in Clay.
```

Comment:
Hey I think in this line should be `authentication` instead of `autenthication` 😅